crates/lib3h/src/gateway/gateway_dht.rs

```diff
@@ -32,7 +32,29 @@ impl<T: Transport, D: Dht> Dht for P2pGateway<T, D> {
 
     /// Processing
     fn post(&mut self, cmd: DhtCommand) -> Lib3hResult<()> {
-        // TODO #179
+        // Add to connection_map for space_gateways
+        // TODO #176 - Maybe we shouldn't have different code paths for populating
+        // the connection_map between space and network gateways.
+        if self.identifier != NETWORK_GATEWAY_ID {
```

Comment:
A little confused about why the network gateway if is constant and how the if statement determines that it's a space gateway. Can you explain this a bit more?

Reply:
P2pGateway code is used for both space and network gateways.
In the RealEngine, there is only one network gateway but potentially many space gateways.
So in P2pGateway code, if the gateway identifier is not the unique network gateway identifier, than its some space gateway.
Ideally we don't want different code behavior between space and network gateways, thats the whole point of this P2pGateway struct, but as of now I do have different behaviors because the connection is handled differently and needs to be sorted out more when implementing the disconnection/reconnection.
Hope thats clear.